src/main/scala/faces/utils/Helpers.scala

```diff
@@ -209,17 +306,37 @@ case class Helpers(cfg: FacesSettings)(implicit rnd: Random) {
     withLight
   }
 
-  def write(img: PixelImage[RGBA], rps: RenderParameter, id: Int, n: Int): Unit ={
-    if (nSamples > 0){
-      PixelImageIO.write(img.map{f=>f.toRGB}, new File(outImgPath + id + "_" + n + ".jpg"))
-      RenderParameterIO.write(rps, new File(outRpsPath + id + "_" + n + ".rps"))
-      writeCSV(rps, new File(outCSVPath + id + "_" + n + ".csv") )
+  def writeImg(img: PixelImage[RGBA], id: Int, n: Int, postfix: String): Unit = {
+    val outImgPathID = outImgPath + id + "/"
+    if (!Path(outImgPathID).exists) {
+      Path(outImgPathID).createDirectory(failIfExists = false)
+    }
+    PixelImageIO.write(img.map { f => f.toRGB }, new File(outImgPathID + id + "_" + n + postfix + ".png"))
+  }
+
+  def writeExceptImage(rps: RenderParameter, id: Int, n: Int): Unit = {
```

Comment:
I do not like this function name. It does not really tell me what this function is about. Difference to writeImg?